src/main/java/lotto_auto/model/LottoResult.java

```diff
@@ -0,0 +1,58 @@
+package lotto_auto.model;
+
+public enum LottoResult {
+
+    FIRST(true, 2000000000L, 1, 6),
+    SECOND(true, 1500000L, 2, 5),
+    THIRD(true, 50000L, 3, 4),
+    FOURTH(true, 5000L, 4, 3),
+    FIFTH(false, 0L, 5, 2),
+    SIXTH(false, 0L, 6, 1),
+    SEVENTH(false, 0L, 7, 0);
+
+    private boolean winning;
+    private long money;
+    private int rank;
+    private int matchNum;
+
+    public boolean isWinning() {
+        return winning;
+    }
+
+    public long getMoney() {
+        return money;
+    }
+
+    public int getRank() {
+        return rank;
+    }
+
+    public int getMatchNum() {
+        return matchNum;
+    }
+
+    LottoResult(boolean winning, long money, int rank, int matchNum) {
+        this.winning = winning;
+        this.money = money;
+        this.rank = rank;
+        this.matchNum = matchNum;
+    }
```

Comment:
getter보다 생성자를 위에 위치시켜주세요!
```suggestion
    LottoResult(boolean winning, long money, int rank, int matchNum) {
        this.winning = winning;
        this.money = money;
        this.rank = rank;
        this.matchNum = matchNum;
    }
    
    public boolean isWinning() {
        return winning;
    }

    public long getMoney() {
        return money;
    }

    public int getRank() {
        return rank;
    }

    public int getMatchNum() {
        return matchNum;
    }
```